packages/react-core/src/components/Nav/__tests__/__snapshots__/Nav.test.tsx.snap

```diff
@@ -5,6 +5,7 @@ exports[`Dark Nav List 1`] = `
   className="test=nav-class"
   onSelect={[Function]}
   onToggle={[Function]}
+  ouiaSafe={true}
```

Comment:
@redallen I'm not sure these should be there......

Reply:
It's okay that these are there. Do you now want them to default to true?

Reply:
well, the attr name is `data-ouia-safe` right - so is this an arg being passed in somewhere?

Reply:
Yep! It's just a prop passed into a component, not the actual DOM rendering.